srcs/parser/check_map_content.c

```diff
@@ -0,0 +1,45 @@
+
+#include <parser.h>
+
+static bool check_valid_char(char c)
+{
+	if (c != ' ' && c != '1' && c != '0'
```

Comment:
Shouldn't we also check for tabs?

Reply:
very true

Reply:
"Spaces are a valid part of themap and are up to you to handle. You must be able to parse any kind of map,as long as it respects the rules of the map." they only mention spaces though so no tabs right?

Reply:
Ah true, I thought the spaces before the lines in the map example they gave were tabs. nvm. this is good.